scripts/jenkins-yoctobuild-build.sh

```diff
@@ -148,7 +148,7 @@ EOF
 
     # Assuming sumo or newer
     cat >> $BUILDDIR/conf/local.conf <<EOF
-EXTERNALSRC_pn-mender = "$WORKSPACE/go"
+EXTERNALSRC_pn-mender-client = "$WORKSPACE/go"
```

Comment:
This will be a problem for building already released Mender versions (for example 2.3.1). The backwards compatible solution is to have here both recipes:
```
EXTERNALSRC_pn-mender = "$WORKSPACE/go"
EXTERNALSRC_pn-mender-client = "$WORKSPACE/go"
```

Reply:
Yes, and please add a comment about it it too, so that when we come back here in two years, maybe we'll remember that the "mender" variant can be removed then.